index.js

```diff
@@ -258,7 +273,7 @@ MochaJUnitReporter.prototype.getTestsuiteData = function(suite) {
 
   var _attr =  {
     name: this._generateSuiteTitle(suite),
-    timestamp: new Date().toISOString().slice(0,-5),
+    timestamp: this._Date.now(),
```

Comment:
Kind of ugly but we need to keep the raw start timestamp and not round it until later. Since we only pass around the XML object, I have to stick it on `timestamp`. Later on, we convert this into a string.

A larger refactoring would pass around domain objects and only structure them as XML in `getXml`, but that's a larger change.